pkg/config/component.go

```diff
@@ -126,3 +142,16 @@ func (c *Component) IsOperatorTest(test *v1.TestInfo) (bool, []string) {
 
 	return false, nil
 }
+
+var namespaceShort = regexp.MustCompile(`ns/(?P<Namespace>[-\w]+)`)
+var namespaceFull = regexp.MustCompile(`namespace/(?P<Namespace>[-\w]+)`)
+
+func ExtractNamespaceFromTestName(in string) string {
+	if namespaceShort.MatchString(in) {
+		return namespaceShort.FindStringSubmatch(in)[1]
+	}
+	if namespaceFull.MatchString(in) {
+		return namespaceFull.FindStringSubmatch(in)[1]
+	}
+	return ""
+}
```

Comment:
A lot of these helpers are already in https://github.com/openshift-eng/ci-test-mapping/blob/main/pkg/util/test_identification.go, would this go there too?